modules/kubernetes/vpa/templates/vpa-values.yaml.tpl

```diff
@@ -1,23 +1,9 @@
 priorityClassName: platform-medium
 
 recommender:
-  # recommender.enabled -- If true, the vpa recommender component will be installed.
-  enabled: true
```

Comment:
Is enabled by default?

Reply:
Si ;)
https://github.com/FairwindsOps/charts/blob/b74f826d24426b97d64081f8344495c0a33dfebd/stable/vpa/values.yaml#L29